test/src/sdk-harness/test_projects/exponentiation/mod.rs

```diff
@@ -0,0 +1,78 @@
+use test_helpers::script_runner;
```

Comment:
Remove this dependency and the associated test for now, it's not ready in the SDK.

Reply:
This test helper is local the stdlib. It has been deprecated though.

Reply:
oops got caught up in the SDK haha

Reply:
Fixed in [712391d](https://github.com/FuelLabs/sway/pull/1744/commits/712391df072afbe63e50765559aae2f1519df015)